mmcls/datasets/pipelines/transforms.py

```diff
@@ -1120,12 +1121,20 @@ def __call__(self, results):
         # dict to albumentations format
         results = self.mapper(results, self.keymap_to_albu)
 
+        if 'gt_label' in results:
```

Comment:
If using backup, this block is useless. please remove it.

Reply:
Done